docs/resources/agent.md

```diff
@@ -0,0 +1,135 @@
+---
```

Comment:
Since it is not possible to create agents from sloctl / terraform on production, we should remove this file (see https://registry.terraform.io/providers/nobl9/nobl9/latest/docs that it is not there)